markdown/solace-masterclass/solace-masterclass.md

```diff
@@ -106,34 +114,47 @@ Once the Solace broker is up and running, click on the broker name and enter it.
+  ![create-mem-popup.png](img/broker-setup/create-mem-popup.png)
+> aside positive Note the environment associated with each MEM that you create. This groups EDA artefacts and their deployments across all brokers associated with a particular environment.
+
+- Enter the newly create MEM and navigate to the **Event Broker Connections** tab and click on the **Connect Event Broker** button as below : \
+![new-mem.png](img/broker-setup/new-mem.png)
+- Select the newly created broker, and you should be able to see a screen as below : \
+![mem-to-broker-assoc.png](img/broker-setup/mem-to-broker-assoc.png)
+> aside positive You will see that :
+> 1. The broker is connected to the MEM
+> 2. The connection between the broker, MEM and the Event Portal is automatically managed by the Event Portal using a cloud hosted Event Management Agent (EMA)
+> 3. To test the connection, click on the **Run Discovery Scan** button which will trigger a scan of your broker
+> 4. The status of the scan is displayed under the **Discovery Scans** section
+
 ## Event Portal design
 
 The Event Portal is a cloud-based tool that simplifies the design of your event-driven architecture (EDA). With the
 Event Portal, you can:
-
 * Define and track relationships between applications in a highly decoupled EDA.
 * Create and manage events using a user-friendly GUI.
-
 In summary, the Event Portal streamlines event management, making it an essential part of your EDA toolkit.
 
 ### Step 1 : Import Event Portal designs
 
 - From the Solace cloud account screen, click on the **Designer** button and enter the Event Portal's Designer page as
   below : ![ep-designer-button.png](img/commons/ep-designer-button.png)
+- Based on the choice of your domain, you can select one of the two domain files as below and download it from the links : \
+      - Banking Industry : [Download Banking Industry App Domain File](https://github.com/SolaceLabs/solace-masterclass-code/blob/main/ep-app-domains/Acme_Bank_App_Domain.json) \
+      - Retail industry : [Download Retail Industry App Domain File](https://github.com/SolaceLabs/solace-masterclass-code/blob/main/ep-app-domains/Acme_Retail_App_Domain.json)
 - Click on the **Import Application Domains** button by clicking on the three dots in the top right corner as below :
   ![ep-app-domain-import.png](img/commons/ep-app-domain-import.png)
 - A file selector dialog box will be displayed so that you can select the application domain file of your choice.
-- Navigate to the folder : `/home/ubuntu/GitHub/solace-masterclass-code/ep-app-domains`
-- As described in **Section 2 - Use case introduction**, you can choose to implement from multiple industry domains.
-  Based on your choice, you can select one of the two domains files as below :
-    - Banking Industry : `Acme_Bank_App_Domain.json`
-    - Retail industry : `Acme_Retail_App_Domain`
-      ![ep-app-domain-files.png](img/commons/ep-app-domain-files.png)
```

Comment:
can you make sure all images no longer used are removed?

Reply:
Done. Removed all the unused images.